lib/core/config.rb

```diff
@@ -11,9 +11,9 @@ class Config # rubocop:disable Metrics/ClassLength
   def initialize(args, options)
     @args = args
     @options = options
-    @org = options[:org]
+    @org = options[:org]&.strip
```

Comment:
Just be aware that we also assign `@org` in line 84 when it comes from `controlplane.yml`:

```
@org = current.fetch(:cpln_org) if current.key?(:cpln_org)
```

I don't know if `current.key?` would account for an empty string.

Just check that and strip if needed. Then feel free to merge, no need to re-request review.

Reply:
Good catch